app/uk/gov/hmrc/incometaxpenaltiesappealsfrontend/views/ReasonableExcusePage.scala.html

```diff
@@ -18,23 +18,29 @@
 @import uk.gov.hmrc.incometaxpenaltiesappealsfrontend.views.ViewUtils.titleBuilder
 @import uk.gov.hmrc.incometaxpenaltiesappealsfrontend.views.html.Layout
 @import uk.gov.hmrc.govukfrontend.views.html.components._
+@import uk.gov.hmrc.incometaxpenaltiesappealsfrontend.views.html.components.ErrorSummary
+@import uk.gov.hmrc.govukfrontend.views.Implicits._
 
 
 @this(layout: Layout,
       govukRadios : GovukRadios,
       formHelper: FormWithCSRF,
+      errorSummary: ErrorSummary,
       govukButton : GovukButton)
 
-@(isLate: Boolean, isAgent: Boolean, form: Form[_])(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
+@(isAgent: Boolean, form: Form[_])(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
 
 @standardOrAgent = {@if(isAgent){agent} else {standard}}
 
-@layout(Some(titleBuilder(messages("reasonableExcuses.headingAndTitle"))), backLinkEnabled = true) {
+@layout(Some(titleBuilder(messages("reasonableExcuses.headingAndTitle"), Some(form))), backLinkEnabled = true) {
+
+    @errorSummary(form, Some("reasonableExcuse"))
```

Comment:
I would change this to use the existing `ErrorSummaryViewModel` like the other pages so that we're consistent:
```suggestion
    @if(form.errors.nonEmpty) {
        @govukErrorSummary(ErrorSummaryViewModel(form))
    }
```